.gitsplit.yml

```diff
@@ -3,173 +3,167 @@ cache_url: "./.gitsplit"
+  - prefix: roles/backup_manager
     target: https://${GITSPLIT_GITHUB_API_TOKEN}@github.com/manala/ansible-role-backup_manager.git
-  - prefix: manala.beanstalkd
+  - prefix: roles/beanstalkd
     target: https://${GITSPLIT_GITHUB_API_TOKEN}@github.com/manala/ansible-role-beanstalkd.git
-  - prefix: manala.bind
+  - prefix: roles/bind
     target: https://${GITSPLIT_GITHUB_API_TOKEN}@github.com/manala/ansible-role-bind.git
-  - prefix: manala.cloud_init
+  - prefix: roles/cloud_init
     target: https://${GITSPLIT_GITHUB_API_TOKEN}@github.com/manala/ansible-role-cloud_init.git
-  - prefix: manala.composer
+  - prefix: roles/composer
     target: https://${GITSPLIT_GITHUB_API_TOKEN}@github.com/manala/ansible-role-composer.git
-  - prefix: manala.cron
+  - prefix: roles/cron
     target: https://${GITSPLIT_GITHUB_API_TOKEN}@github.com/manala/ansible-role-cron.git
-  - prefix: manala.deploy
+  - prefix: roles/deploy
     target: https://${GITSPLIT_GITHUB_API_TOKEN}@github.com/manala/ansible-role-deploy.git
-  - prefix: manala.dhcp
+  - prefix: roles/dhcp
     target: https://${GITSPLIT_GITHUB_API_TOKEN}@github.com/manala/ansible-role-dhcp.git
-  - prefix: manala.dnsmasq
+  - prefix: roles/dnsmasq
     target: https://${GITSPLIT_GITHUB_API_TOKEN}@github.com/manala/ansible-role-dnsmasq.git
-  - prefix: manala.docker
+  - prefix: roles/docker
     target: https://${GITSPLIT_GITHUB_API_TOKEN}@github.com/manala/ansible-role-docker.git
-  - prefix: manala.elasticsearch
+  - prefix: roles/elasticsearch
     target: https://${GITSPLIT_GITHUB_API_TOKEN}@github.com/manala/ansible-role-elasticsearch.git
-  - prefix: manala.environment
+  - prefix: roles/environment
     target: https://${GITSPLIT_GITHUB_API_TOKEN}@github.com/manala/ansible-role-environment.git
-  - prefix: manala.fail2ban
+  - prefix: roles/fail2ban
     target: https://${GITSPLIT_GITHUB_API_TOKEN}@github.com/manala/ansible-role-fail2ban.git
-  - prefix: manala.files
+  - prefix: roles/files
     target: https://${GITSPLIT_GITHUB_API_TOKEN}@github.com/manala/ansible-role-files.git
-  - prefix: manala.git
+  - prefix: roles/git
     target: https://${GITSPLIT_GITHUB_API_TOKEN}@github.com/manala/ansible-role-git.git
-  - prefix: manala.gitlab
+  - prefix: roles/gitlab
     target: https://${GITSPLIT_GITHUB_API_TOKEN}@github.com/manala/ansible-role-gitlab.git
-  - prefix: manala.glusterfs
+  - prefix: roles/glusterfs
     target: https://${GITSPLIT_GITHUB_API_TOKEN}@github.com/manala/ansible-role-glusterfs.git
-  - prefix: manala.grafana
+  - prefix: roles/grafana
     target: https://${GITSPLIT_GITHUB_API_TOKEN}@github.com/manala/ansible-role-grafana.git
-  - prefix: manala.graylog_sidecar
+  - prefix: roles/graylog_sidecar
     target: https://${GITSPLIT_GITHUB_API_TOKEN}@github.com/manala/ansible-role-graylog_sidecar.git
-  - prefix: manala.haproxy
+  - prefix: roles/haproxy
     target: https://${GITSPLIT_GITHUB_API_TOKEN}@github.com/manala/ansible-role-haproxy.git
-  # Deprecated
```

Comment:
It doesn't eat of bread to update deprecate roles too :)

Reply:
done